python/packages/autogen-core/src/autogen_core/components/models/_openai_client.py

```diff
@@ -393,18 +401,36 @@ async def create(
 
         if len(tools) > 0:
             converted_tools = convert_tools(tools)
-            future = asyncio.ensure_future(
-                self._client.chat.completions.create(
-                    messages=oai_messages,
-                    stream=False,
-                    tools=converted_tools,
-                    **create_args,
+            if self._use_beta_client:
+                future = asyncio.ensure_future(
+                    self._client.beta.chat.completions.parse(
```

Comment:
What other ramifications does `parse` vs `create` have in terms of the API call?

Reply:
parse is the recommended approach moving forward based on this cookbook - https://cookbook.openai.com/examples/structured_outputs_intro#using-the-sdk-parse-helper

is there an easy way to test the difference between these 2 approaches?

Reply:
Recommended approach for when using structured outputs right? If you just want completions then is `parse` the call you should be making?

It seems to me that they are completely different APIs?

Reply:
structured output for sure, but there is no `create` option for completions in beta SDK
https://github.com/openai/openai-python/blob/8467d41376c2b17eae1c78d56b39767e7807cb6f/src/openai/resources/beta/chat/completions.py#L59